key.go

```diff
@@ -185,6 +193,14 @@ const (
 	KeyCtrlCaret        KeyType = keyRS  // ctrl+^
 	KeyCtrlUnderscore   KeyType = keyUS  // ctrl+_
 	KeyCtrlQuestionMark KeyType = keyDEL // ctrl+?
+	KeyCtrlUp           KeyType = KeyCU  // ctrl+up
+	KeyCtrlDown         KeyType = KeyCD  // ctrl+down
+	KeyCtrlRight        KeyType = KeyCR  // ctrl+right
+	KeyCtrlLeft         KeyType = KeyCL  // ctrl+left
+	KeyShiftUp          KeyType = KeySU  // shift+up
+	KeyShiftDown        KeyType = KeySD  // shift+down
+	KeyShiftRight       KeyType = KeySR  // shift+right
+	KeyShiftLeft        KeyType = KeySL  // shift+left
```

Comment:
These can be moved to the "other keys" block below and you can just have the values auto-set via the iota.

Reply:
If I do this, then for me, "msg.String()" or just `fmt.Println(msg)` is just totally empty.
So key.NewBinding ... WithKeys = .. nothing, because its empty, nothing t o match against.. and switching just msg.String also impossible as no string.

Reply:
Done